core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -527,29 +542,6 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
 
       pendingReplayTasksCount.addAndGet(tasks.size)
 
-      // Wait for all tasks to finish. This makes sure that checkForLogs
-      // is not scheduled again while some tasks are already running in
-      // the replayExecutor.
```

Comment:
It would be good to move this comment to `processing`.

Reply:
done